.github/workflows/build.yml

```diff
@@ -41,6 +41,31 @@ jobs:
         run: bundle-audit check --update
       - name: Build inspec_tools gem
         run: gem build inspec_tools.gemspec
+      - uses: actions/upload-artifact@v2
+        with:
+          name: inspec_tools_gem
```

Comment:
if you are going to use a matrix, this artifact name will be overwritten by each potential matrix and needs to be parameterized to include the matrix information and then download then needs to be changed to download the appropriate artifact for the matrix.

Reply:
Yes. I don't know if build should be ran for each platform since then we would need to test each built gem that way. That would make our 1 build and 3 tests turn into 3 builds and 9 tests. I have a good idea on how to make this work though.